tests/lib/rules/no-shadow-restricted-names.js

```diff
@@ -89,6 +89,21 @@ ruleTester.run("no-shadow-restricted-names", rule, {
                 { message: "Shadowing of global property 'eval'.", type: "Identifier" },
                 { message: "Shadowing of global property 'eval'.", type: "Identifier" }
             ]
+        },
```

Comment:
could you please add some more tests like:
```js
var undefined = "foo";
var foo = bar, undefined = bar;
```

Reply:
What's the different between the existing tests?

Reply:
oh, my apologies! I didn't notice it was already there!